Assets/Scripts/Models/Need.cs

```diff
@@ -33,18 +30,21 @@ private Need(Need other)
         Type = other.Type;
         LocalisationID = other.LocalisationID;
         Name = other.Name;
-        growthRate = other.growthRate;
+        GrowthRate = other.GrowthRate;
         highToLow = other.highToLow;
         RestoreNeedFurn = other.RestoreNeedFurn;
         RestoreNeedTime = other.RestoreNeedTime;
-        restoreNeedAmount = other.restoreNeedAmount;
-        CompleteOnFail = other.CompleteOnFail;
-        addedInVacuum = other.addedInVacuum;
-        dps = other.dps;
-        luaUpdate = other.luaUpdate;
-        luaOnly = other.luaOnly;
+        RestoreNeedAmount = other.RestoreNeedAmount;
+        Damage = other.Damage;
+
+        if (other.EventActions != null)
```

Comment:
This being a project involving lots of people, I'd prefer to go the defensive coding route, to ensure things don't crash unnecessarily if someone later does something unexpected that leads to it being called before EventActions exists, or some other thing I'm not thinking about ahead of time.